Source/WebInspectorUI/UserInterface/Base/Main.js

```diff
@@ -1825,11 +1844,27 @@ WI._updateWindowInactiveState = function(event)
 
 WI._windowResized = function(event)
 {
+    WI._updateLayoutMode();
     WI.tabBar.updateLayout(WI.View.LayoutReason.Resize);
-    WI._tabBrowserSizeDidChange();
     WI._updateSheetRect();
 };
 
+WI._updateLayoutMode = function()
+{
+    let computedMode = WI.LayoutMode.Default;
+    
+    if (WI.settings.enableNarrowLayoutMode.value && window.innerWidth < WI.NarrowLayoutMaximumWidth)
```

Comment:
so i did a bit of testing and there's a brief period right before you hit `WI.NarrowLayoutMaximumWidth` where the minima widths of the navigation sidebar and main content area cause the details sidebar to get clipped outside the bounds of the Web Inspector area.  i think we may want to improve this by not only checking `window.innerWidth < WI.NarrowLayoutMaximumWidth` but also if there's even enough space to show everything on one line (or alternatively increasing `WI.NarrowLayoutMaximumWidth` to be equal to the `min-width` of the `#navigation-sidebar` and `#tab-browser` and `#details-sidebar`).  ideally this would be fixed as part of this PR, but imo it'd be ok to address as a followup since it's not technically directly relevant to the goal of this PR

Reply:
I've made the width wider for now. IMO it's more intuitive code-wise to have a single breakpoint that triggers narrow mode rather than more complex set of factors (and since it's checked every time the user resizes, ig it's slightly more efficient too, though if you've proposing the other solution it can't be too big a difference).